internal/endpoints/github.go

```diff
@@ -24,11 +23,8 @@ func (eh *EndpointHandler) GithubWebhook(w http.ResponseWriter, r *http.Request)
 
 	services := config.Get().Services
 
-	if config.Get().Debug {
-		payload, err = ioutil.ReadAll(r.Body)
-	} else {
-		payload, err = github.ValidatePayload(r, []byte(config.Get().GithubWebhookSecretKey))
-	}
+	payload, err = github.ValidatePayload(r, []byte(config.Get().GithubWebhookSecretKey))
```

Comment:
We should keep the debug logic just so we can rule out validator issues in the future